src/frontend/staff-portal/src/app/components/jj-workbench/jj-dispute-hearing-inbox/jj-dispute-hearing-inbox.component.html

```diff
@@ -5,7 +5,7 @@ <h3><b>Hearing Inbox</b> ({{dataSource.filteredData.length}} results)</h3>
   </div>
   <div>
     <input matInput [matDatepicker]="picker" [formControl]="appearanceDateFilter" (click)='picker.open()'
-      placeholder="court date" style="background-color: #f2f2f2;"/>
+      placeholder="court date" [readonly]="true" style="background-color: #f2f2f2;"/>
```

Comment:
this is not a readonly field, users need to be able to change the input as well as use the datepicker

Reply:
It can still activate the datepicker by clicking. readonly input only affect the date can be modify by typing or not